### ISSUE TITLE
Conda install astroplan

### ISSUE THREAD

**Issue by bmorris3** [astropy/astroplan#249]
Hey @mwcraig,

In the [astroplan installation docs](https://github.com/astropy/astroplan/blob/master/docs/installation.rst#installation-1), we suggest the line for conda installing astroplan is: 
```
conda install -c https://conda.binstar.org/astropy astroplan
```
Can/should that line be abbreviated to: 
```
conda install -c astropy astroplan
```
Thanks!

**Comment by mwcraig** [comment]
Yes, the abbreviated form should be more stable over the long run (in case anaconda changes the URL for the repositories)

**Comment by bmorris3** [comment]
Fixed by #250.
